### PR TITLE
Un-graingert copyright notice

### PR DESCRIPTION
**Pull request by graingert**
Add woodruffw to copyright notice

### CODE REVIEW

**Comment by graingert** [comment]
@woodruffw not sure what's best here - I can't just unilaterally add you as a copyright holder. Maybe this should be "Copyright Contributors to the Zimzmor project."

**Comment by woodruffw** [comment]
> Maybe this should be "Copyright Contributors to the Zimzmor project."

Yeah, that works for me!

**Comment by graingert** [comment]
sorry not Zimzmor, I mean zizmor

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2024 Thomas Grainger.
+Copyright (c) 2024 Thomas Grainger and William Woodruff <william @ yossarian.net>
```
Comment (graingert) [inline LICENSE:3]:
```suggestion
Copyright (c) 2024 Copyright Contributors to the zizmor project.
```

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2024 Thomas Grainger.
+Copyright (c) 2024 Copyright Contributors to the zizmor project.
```
Comment (graingert) [inline LICENSE:3]:
```suggestion
Copyright (c) 2024 Contributors to the zizmor project.
```

**Review by woodruffw — APPROVED** [review]

**Pull request merged**